.travis/publish_docs.sh

```diff
@@ -67,13 +80,14 @@ else
 fi
 
 printf "${CYAN}Generating site.$RESET \n"
-mvn clean site
+mvn clean site > /dev/null
```

Comment:
otherwise this will fill the logs and will fail the build. It happened several times when I was trying different things